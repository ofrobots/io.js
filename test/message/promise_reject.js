'use strict';
const common = require('../common');

// Flags: --expose-gc

new Promise(function(res, rej) {
  consol.log('oops'); // eslint-disable-line no-undef
});

// Manually call GC due to possible memory contraints with attempting to
// trigger it "naturally".
setTimeout(common.mustCall(() => {
  gc();
  gc();
  gc();
}, 1), 2);
