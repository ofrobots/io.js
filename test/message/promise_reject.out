*test*message*promise_reject.js:*
  consol.log('oops'); // eslint-disable-line no-undef
  ^

ReferenceError: consol is not defined
    at *test*message*promise_reject.js:*:*
    at Object.<anonymous> (*test*message*promise_reject.js:*:*)
    at Module._compile (module.js:*:*)
    at Object.Module._extensions..js (module.js:*:*)
    at Module.load (module.js:*:*)
    at tryModuleLoad (module.js:*:*)
    at Function.Module._load (module.js:*:*)
    at Function.Module.runMain (module.js:*:*)
    at startup (node.js:*:*)
    at node.js:*:*
